virt/virt-4-9-release-notes.adoc

```diff
@@ -41,6 +41,7 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 //CNV-8875 OLM and stable channels
 
 //CNV-8577 OpenShift Service Mesh
+* {VirtProductName} is now integrated with OpenShift Service Mesh. You can xref:../virt/virtual_machines/vm_networking/virt-connecting-vm-to-service-mesh.adoc#virt-connecting-vm-to-service-mesh[connect virtual machines to a service mesh] to monitor, visualize, and control traffic between pods that run virtual machine workloads on the default pod network with IPv4.
```

Comment:
The second sentence is a little bit long, but I think you probably phrased it the best possible way.